src/main/java/io/swagger/inflector/utils/ReflectionUtils.java

```diff
@@ -239,12 +239,13 @@ public JavaType getTypeFromModel(String name, Model model, Map<String, Model> de
         if(model instanceof ArrayModel) {
             ArrayModel am = (ArrayModel) model;
             Property inner = am.getItems();
-            JavaType innerType = getTypeFromProperty(inner.getType(), inner.getFormat(), inner, definitions);
-            if(innerType != null) {
-                return tf.constructArrayType(innerType);
-            }
-            else {
-                return tf.constructArrayType(JsonNode.class);
+            if(inner != null) {
```

Comment:
A NPE was being caused by malformed swagger docs where array items was not set..